src/cli/cli_config.h

```diff
@@ -35,6 +35,8 @@
 #ifndef CONFIG_CLI_H_
 #define CONFIG_CLI_H_
 
+#include "openthread-core-config.h"
+
 #include <openthread/config.h>
```

Comment:
We can consider removing this since it would be included by `core-config` header
```suggestion
```

Reply:
Updated